metrics/resource_view.go

```diff
@@ -313,6 +358,7 @@ var _ view.Meter = (*defaultMeterImpl)(nil)
 var defaultMeter = meterExporter{
 	m: &defaultMeterImpl{},
 	o: stats.WithRecorder(nil),
+	t: cleanupClock.Now(),
```

Comment:
Why does this need a clock?

Reply:
It doesn't need a clock - it needs a timestamp.  However, I just realized that we can avoid ever reading this for the defaultMeter, since it never expires.